src/components/SyncIndicator.tsx

```diff
@@ -184,181 +173,248 @@ export const SyncIndicator = ({ handle }: { handle: DocHandle<unknown> }) => {
                 )}
               </dd>
             </div>
+            {headsView}
           </dl>
         </PopoverContent>
       </Popover>
     );
   }
 };
 
-const SYNC_SERVER_PREFIX = "storage-server-";
+const SYNC_SERVER_STORAGE_ID =
+  "3760df37-a4c6-4f66-9ecd-732039a9385d" as StorageId;
```

Comment:
does this mean it only works with the deployed sync.automerge.org server and not a localhost server?
if so, can we make this an environment variable and fallback to this as a default?

Reply:
yes we currently don't have a good mechanism to get the storage id of the sync server dynamically. I will fix this